src/backends/backend.js

```diff
@@ -26,7 +26,24 @@ class LocalStorageAuthStore {
 
 const slugFormatter = (template = "{{slug}}", entryData) => {
   const date = new Date();
-  const identifier = entryData.get("title", entryData.get("path"));
+
+  const getIdentifier = (entryData) => {
+	  const validIdentifierFields = ["title", "path"];
+	  const identifiers = validIdentifierFields.map((field) => {
+	  	return entryData.find((_, key) => {
+	  		return key.toLowerCase() === field;
+	  	});
+	  });
+	  
+	  const identifier = identifiers.find(i => typeof i !== 'undefined');
+
+	  if (typeof identifier === 'undefined') {
```

Comment:
`Array.find` always returns `undefined` when it doesn't match (per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find), and `undefined` always `=== undefined`, so this comparison can be changed to the simpler `identifier === undefined` (`identifier` is always a defined variable, which is what the `typeof identifier === undefined` would normally be used to detect).